notesapi/v1/search_indexes.py

```diff
@@ -12,6 +12,7 @@ class NoteIndex(indexes.SearchIndex, indexes.Indexable):
     created = indexes.DateTimeField(model_attr='created')
     updated = indexes.DateTimeField(model_attr='updated')
     tags = indexes.CharField(model_attr='tags')
+    data = indexes.CharField(use_template=True)
```

Comment:
Does this mean we have to recreate the search index (like I had to when I added the tags field right above)?